### DISCUSSION TITLE
ROG STRIX RTX 3090 Fan Connect II support

### DISCUSSION THREAD

**Discussion by VeilSide47** [Rem0o/FanControl.Releases#1407] [Q&A]
Hello, I have ROG STRIX RTX 3090 and was wondering if it's possible to control external PWM fans connected to GPU's Fan Connect II headers. I don't want to use Asus rubbish GPU Tweak III app which half of the time doesn't even work.

**Comment by bjdiss** [comment]
According to the Asus docs, those fan headers follow the GPU fan speed and are not independently controllable.

**Reply by VeilSide47** [reply]
Unfortunately they do not follow anything unless you have their proprietary app installed. It's ridiculous you need an app just to enable them external fans.
